src/DI/Container.php

```diff
@@ -163,4 +163,57 @@ public function http() {
 	public function event() {
 		return $this['ilAppEventHandler'];
 	}
+
+	/**
+	 * @return \ilIniFile
+	 */
+	public function iliasIni() {
```

Comment:
I'd propose to have something like `$DIC->config()->client()` and `$DIC->config()->app()`, what do you think?